--- ./configure.in.org	2007-03-29 15:35:25.000000000 +0200
+++ ./configure.in	2007-04-20 16:09:51.000000000 +0200
@@ -82,16 +82,6 @@
 PHP_LDFLAGS=$($PHP_CONFIG --libs)
 AC_SUBST(PHP_INCLUDES)
 AC_SUBST(PHP_LDFLAGS)
-LDFLAGS+=" -L$($PHP_CONFIG --prefix)/lib"
-
-# Checks for libraries.
-AC_CHECK_LIB([php5], [php_module_startup],[], [have_php5="no"], [$PHP_LDFLAGS])
-if test "$have_php5" = "no"; then
-    AC_CHECK_LIB([php4], [php_module_startup], [], [have_php4="no"], [$PHP_LDFLAGS])
-fi
-if test "$have_php5" = "no" -a "$have_php4" = "no"; then
-    AC_MSG_ERROR([Cannot locate a proper php library])
-fi
 
 
 AC_CONFIG_FILES([Makefile])
